sparsity/sparse_frame.py

```diff
@@ -243,6 +294,25 @@ def groupby(self, by=None, level=0):
         return self.groupby_sum(by, level)
 
     def groupby_agg(self, by=None, level=None, agg_func=None):
+        """ Aggregate data using callable.
+
+        Parameters
+        ----------
+        by: array-like, string
+            grouping array or grouping column as string
+        level: int
+            which level from index to use if multiindex
+        agg_func: callable
+            Function which are applied to groups. Should
+            work on scipy.sparse_csr_matrizes and return
```

Comment:
Since it's a documentation PR, I can be picky about words. :P
You always pluralise words ending with "x" with "izes": "matrizes", "indizes", while it should be "matrices" and "indices". :)